source/connection/network-compression.txt

```diff
@@ -1,95 +1,102 @@
-.. _compression:
+.. _java-compression:
```

Comment:
I would suggest including network in the name; `java-compression` seems a generic expression including compression of text, image, sound, etc.